modules/ROOT/pages/upgrade-cluster.adoc

```diff
@@ -4,11 +4,11 @@ Anypoint Runtime Fabric is powered by a set of components including Docker, Kube
 
 == Before You Begin
 
-* Run the upgrade procedure on a non-production Runtime Fabric to verify compatibility with your environment and to gain familiarity with the procedure.
+* Run the upgrade procedure on a nonproduction instance of Runtime Fabric to verify compatibility with your environment and to gain familiarity with the procedure.
```

Comment:
"Nonproduction" (non-hyphenated) looks odd to me; I don't think that's consistent with other areas of our docs.

Reply:
Favor keeping "Runtime Fabric" rather than "instance of.."